cogs/custom.py

```diff
@@ -25,6 +27,25 @@ async def ping(self, ctx):
         duration = (end - start) * 1000
         await message.edit(embed=info(f":ping_pong: {duration:.2f}ms", ctx.me, "Pong!"))
 
+    @commands.command(aliases=("poll",))
+    async def vote(self, ctx: commands.Context, title: str, *options: str) -> None:
+        """
+        Build a quick voting poll with matching reactions with the provided options.
+        A maximum of 20 options can be provided, as Discord supports a max of 20
+        reactions on a single message.
```

Comment:
```suggestion
        Build a quick voting poll with matching reactions with the provided options.
        
        A maximum of 20 options can be provided, as Discord supports a max of 20
        reactions on a single message.
```
Multiline docstrings should consist of a "title" line separated from the rest of the docstring which is considered as "description"